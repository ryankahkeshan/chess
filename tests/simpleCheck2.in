game human human
move f2 f4
move e7 e5
move f4 e5
move d8 h4
resign
