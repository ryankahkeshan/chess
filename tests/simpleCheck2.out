8 rnbqkbnr
7 pppppppp
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2 PPPPPPPP
1 RNBQKBNR

  abcdefgh

8 rnbqkbnr
7 pppppppp
6  _ _ _ _
5 _ _ _ _ 
4  _ _ P _
3 _ _ _ _ 
2 PPPPP_PP
1 RNBQKBNR

  abcdefgh

8 rnbqkbnr
7 pppp_ppp
6  _ _ _ _
5 _ _ p _ 
4  _ _ P _
3 _ _ _ _ 
2 PPPPP_PP
1 RNBQKBNR

  abcdefgh

8 rnbqkbnr
7 pppp_ppp
6  _ _ _ _
5 _ _ P _ 
4  _ _ _ _
3 _ _ _ _ 
2 PPPPP_PP
1 RNBQKBNR

  abcdefgh

8 rnb_kbnr
7 pppp_ppp
6  _ _ _ _
5 _ _ P _ 
4  _ _ _ q
3 _ _ _ _ 
2 PPPPP_PP
1 RNBQKBNR

  abcdefgh

White is in check.
Black wins!
Final Score:
White: 0
Black: 1
